cli/util/util.go

```diff
@@ -938,3 +940,31 @@ func CopyFileDeep(src, dst string) error {
 	}
 	return copy.Copy(src, dst)
 }
+
+// StringToTimestamp transforms string with number or RFC339Nano time
+// to <sec.nanosec> timestamp string.
+func StringToTimestamp(input string) (string, error) {
+	if input == "" {
+		// Default value.
+		return strconv.FormatUint(math.MaxUint64, 10), nil
+	}
+
+	// Float.
+	floatTimestamp, err := strconv.ParseFloat(input, 64)
```

Comment:
I think that this comment is not so informative due to we have self-explanatory function on the next line.

Reply:
Agree.